controller/tap/handlers.go

```diff
@@ -330,6 +330,7 @@ func mkPathItem(desc string) spec.PathItem {
 							},
 						},
 					},
+					ID: "tapResourceV0",
```

Comment:
Nice fix @kleimkuhler 

Small nit: I'd put this above `Responses`, just to make it a bit more readable, but TIOLI 👍

Reply:
forgot to mention that this works for me locally :)